client/lib/analytics/ad-tracking/google-analytics.js

```diff
@@ -38,15 +38,25 @@ export function getGoogleAnalyticsDefaultConfig() {
  * @param {string} urlPath The path of the current page
  * @param {string} pageTitle The title of the current page
  * @param {boolean} useJetpackGoogleAnalytics send the page view to Jetpack Google Analytics
+ * @param {boolean} useAkismetGoogleAnalytics send the page view to Akismet Google Analytics
  */
 export function fireGoogleAnalyticsPageView(
 	urlPath,
 	pageTitle,
-	useJetpackGoogleAnalytics = false
+	useJetpackGoogleAnalytics = false,
```

Comment:
as we grow our list of exceptions, we could consider creating an object with `use(...)GoogleAnalytics`? I can see that it could be out of the scope for that PR - but maybe let's create a followup maintenance task to address it later?

Reply:
Yes I agree with this. I won't hold up this PR going out, I'll add it to maintenance